srcDoc/release_notes.rst

```diff
@@ -80,6 +80,13 @@ Added RTI Perftest and RTI Connext DDS information at beginning of the test (#54
 Starting with this release, RTI Perftest will print at the beginning of the test
 its version and the version of RTI Connext DDS used to compile against.
 
+Update `qos_string.h` with the value of `perftest_qos_profiles.xml` in the build step(#63)
```

Comment:
Add space

Reply:
Automatically regenerate `qos_string.h` files if `perftest_qos_profiles.xml` is modified

Reply:
Done